recipe/patch_yaml/pandas.yaml

```
@@ -17,3 +17,47 @@ then:
   - replace_depends:
       old: pytz >=2017.2
       new: pytz >=2020.1
+
+---
+
+if:
+  name: pandas
+  version: 2.2.0
+  timestamp_lt: 1752072645000
+then:
+  - replace_depends:
+      old: numpy >=1.19
+      new: numpy >1.22.3
```

Let's match the feedstock
```suggestion
      new: numpy >=1.22.4
```